### PR TITLE
Added Xcode 10.2 support, add 10.2 to Travis matrix

### PR DESCRIPTION
**Pull request by timdonnelly**

### CODE REVIEW

File: swift/Workflow/Tests/WorkflowNodeTests.swift [hunk 1]
```diff
@@ -179,6 +179,8 @@ final class WorkflowNodeTests: XCTestCase {
             struct State {}
 
             typealias Output = Int
+            
+            typealias Rendering = Void
```
Comment (timdonnelly) [inline swift/Workflow/Tests/WorkflowNodeTests.swift]: It appears that the Swift 5 compiler (even when running in 4.2 mode) has different behavior when it comes to type inference.

**Comment by bencochran** [comment]
Should we add Swift 5 support to the podspec and Package.swift?

**Comment by timdonnelly** [comment]
> Should we add Swift 5 support to the podspec and Package.swift?

I would, but `.v5` is only available when using SPM 5+ – so you always have to build with the latest version of the tooling.

**Comment by bencochran** [comment]
> I would, but `.v5` is only available when using SPM 5+

Cool cool. I tried to find the answer to that very question documented somewhere and had no luck (same with source code, it wasn’t immediately obvious what version shipped with what definitions).

**Pull request merged**